tools/hilbert.py

```diff
@@ -675,6 +675,41 @@ def cmd_app_change(parser, context, args):
 
     return args
 
+@subcmd('cfg_query', help="query configuration properties")
+def cmd_cfg_query(parser, context, args):
+    action = 'cfg_query'
+    log.debug("Running 'cmd_{}'".format(action))
+
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument('--configfile', required=False,
+                       help="specify input .YAML file (default: 'Hilbert.yml')")
+    group.add_argument('--configdump', required=False,
+                       help="specify input dump file")
+
+    parser.add_argument('QueryString', help="specify the query")
+
+    args = parser.parse_args(args)
+
+    log.debug("Querying '{}' from the configuration...".format(args.QueryString))
+
+    obj = None
+    try:
+        obj = cmd_list(parser, context, args, args.QueryString)
+    except:
+        log.exception("Sorry could not get the result of query '{}' from the input file!".format(args.QueryString))
+        sys.exit(1)
+
+    assert obj is not None
+
+    if isinstance(obj, BaseValidator):
+        yaml_dump(obj.data_dump(),sys.stdout)
+    elif isinstance(obj, (str, int, float, complex)):
+        print(obj)
+    else:
+        yaml.dump(obj,sys.stdout)
```

Comment:
Same here. `yaml.dump("test")` prints
```
test
...
```
`yaml.dump(dict(a=1))` prints
```
{a: 1}

```
with an empty newline. So I'd say it's not even consitent!?

Reply:
try to use `yaml_dump` instead